rules/enforce-store-naming-convention/enforce-store-naming-convention.js

```diff
@@ -7,19 +7,26 @@ module.exports = {
     type: "problem",
     docs: {
       description:
-        "Enforce $ as a prefix for any store created by Effector methods",
+        "Enforce $ as a prefix or postfix for any store created by Effector methods",
       category: "Naming",
       recommended: true,
     },
     messages: {
       invalidName:
-        'Store "{{ storeName }}" should be named with prefix, rename it to "${{ storeName }}"',
-      renameStore: 'Rename "{{ storeName }}" to "${{ storeName }}"',
+        'Store "{{ storeName }}" should be named with {{ mode }}, rename it to "{{ correctedStoreName }}"',
+      renameStore: 'Rename "{{ storeName }}" to "{{ correctedStoreName }}"',
     },
-    schema: [],
+    schema: [
+      {
+        "enum": ["prefix", "postfix"]
+      }
+    ],
   },
   create(context) {
-    const parserServices = context.parserServices;
+    const { parserServices, options } = context;
+    // prefix mode is a default option
+    const [mode = "prefix"] = options;
```

Comment:
May we validate input here? What if user write incorrect mode? I think, we should raise a readable exception in this case.

Reply:
I added this, but not sure if the message is clear 😄 Do you have something on your mind?

Reply:
I like the message, but I think, we should add more information — like "the configuration of effector-plugin-eslint ..."